NineChronicles.Standalone/GraphTypes/ValidationQuery.cs

```diff
@@ -25,22 +25,25 @@ public ValidationQuery(StandaloneContext standaloneContext)
                     var raw = context.GetArgument<string>("raw");
                     try
                     {
+                        Log.Debug($"Validating received raw: {raw}");
                         // FIXME: Thread.Sleep is temporary. Should be removed.
                         var timeSpent = 0;
                         var retryInterval = 1000;
+                        const int grace = 100 * 1000;
                         while (standaloneContext.BlockChain is null)
                         {
                             Log.Debug("Blockchain instance is null. Sleep...");
                             Thread.Sleep(retryInterval);
                             timeSpent += retryInterval;
 
-                            if (timeSpent < 100 * 1000)
+                            if (timeSpent < grace)
                             {
                                 continue;
                             }
-                            
-                            Log.Debug("Blockchain instance is still null.");
-                            return true;
+
+                            var msg = $"Blockchain instance is not initialized until {grace}.";
+                            Log.Debug(msg);
+                            throw new NullReferenceException(msg);
```

Comment:
We may need `BlockChainInitializeError` instead of just `NullReferenceException`.